### DISCUSSION TITLE
Transparency not working Windows 11

### DISCUSSION THREAD

**Discussion by Eddyphish** [Open-Shell/Open-Shell-Menu#1837] [Q&A]
Hey,

I love this menu but am I being dumb or is the transparency not really working here? I've tried it with full transparency, two-tone, glass, changed the opacity... both the taskbar and start menu look opaque regardless. I'm running Windows 11 (sadly). 

Any help appreciated! Screenshots below.

![Screenshot 2024-02-10 121005](https://github.com/Open-Shell/Open-Shell-Menu/assets/159624979/88a3e2a9-1769-4e79-a29e-6fc088c6ed2e)
![Screenshot 2024-02-10 121034](https://github.com/Open-Shell/Open-Shell-Menu/assets/159624979/b1201bc8-c7b5-44b0-ae87-c0445cffa7d6)

**Comment by ge0rdi** [comment]
Hello.

Taskbar customizations don't work since Windows 11 22H2.
See #1342 for more details.

You can use `TranslucentTB` if you want taskbar transparency.

**Reply by MidnightInAurora** [reply]
That's the solution?  Use something else?

I had the transparency issue on Win 11 Ent and stopped by here assuming that the distro I had was simply out of date.  It wasn't.

Open-shell hasn't has a release since Aug/23 and there's 460 issues pending.

If this is a dead product, just say so.  If not, if it's something you want people to view as a first-choice product, then GET OFF YOUR COLLECTIVE ASSES AND MAKE IT HAPPEN.

Jim
